### DISCUSSION TITLE
LFX Mentorship '23 Term 03 September - November: Call for Proposals

### DISCUSSION THREAD

**Discussion by nate-double-u** [cncf/mentoring#1032] [Announcements]
Hello everyone,

The LFX Mentorship 2023 Term 03 Sept - Nov is open for project ideas:
https://github.com/cncf/mentoring/tree/main/programs/lfx-mentorship/2023/03-Sep-Nov

Project submission and application timeline:
* project proposals due: **July 27**, 2023
* mentee applications open: **Aug 2 - 15**, 2023
* application review/admission decisions/HR paperwork: **Aug 16 - 29**, 2023

If your Google Summer of Code proposal was declined and you didn't participate in LFX Term 2, this is still a good alternative -- proposals can be copied over almost 1:1.

We're looking forward to seeing all the project ideas you're interested in working on!

### Timeline

Mentorship duration - three months (12 weeks - full-time schedule)

| activity | date |
| --- | --- |   
| project proposals | Thur July 27, 5:00 PM PDT |
| mentee applications open | Wed Aug 2 - Tues 15, 5:00 PM PDT |
| application review/admission decisions | Wed Aug 16 - Tues Aug 29, 5:00 PM PDT |
| Mentorship program begins with the initial work assignments | Mon Sept 4 (Week 1) | 
| Midterm mentee evaluations and first stipend payments | Wed Oct 11 (Week 6) |
| Final mentee evaluations and mentee feedback/blog submission due, second and final stipend payment approvals | Wed Nov 22, 5:00 PM PST (Week 12) |
| Last day of term | Thur Nov 30 |

---

[Announcement email](https://lists.cncf.io/g/tag-cs-mentoring-wg/message/10)

**Comment by satyampsoni** [comment]
when will the project be listed?

**Reply by aliok** [reply]
hi @satyampsoni 

Mentors will start proposing projects for LFX Mentorship now and they will be able to do that until July 27. 
You can watch https://github.com/cncf/mentoring/blob/main/programs/lfx-mentorship/2023/03-Sep-Nov/project_ideas.md for new proposals added.

In August 2, mentees will be able to apply using the LFX Mentorship website.
